storage/gc_queue_test.go

```diff
@@ -349,7 +355,9 @@ func TestGCQueueIntentResolution(t *testing.T) {
 	// Iterate through all values to ensure intents have been fully resolved.
 	meta := &engine.MVCCMetadata{}
 	err := tc.store.Engine().Iterate(engine.MVCCEncodeKey(proto.KeyMin), engine.MVCCEncodeKey(proto.KeyMax), func(kv proto.RawKeyValue) (bool, error) {
-		if key, _, isValue := engine.MVCCDecodeKey(kv.Key); !isValue {
+		if key, _, isValue, err := engine.MVCCDecodeKey(kv.Key); err != nil {
+			t.Fatal(err)
```

Comment:
Yeah. Not sure if that buys us much (same number of lines of code), but it is done.